### PR TITLE
Add docker support

### PR DESCRIPTION
**Pull request by vdbg**

### CODE REVIEW

File: collector.py [hunk 1]
```diff
@@ -13,19 +13,29 @@ def __init__(self) -> None:
         self.devices_rssi: dict[str, int] = {}
 
     def run(self) -> None:
-        result = subprocess.run(['sudo', 'btmgmt', 'find'], stdout=subprocess.PIPE)
+        # Piping 'hello' to stdin because otherwise btmgmt will silently fail in docker containers
+        # https://www.spinics.net/lists/linux-bluetooth/msg85222.html
+        result = subprocess.run(['sudo', 'btmgmt', 'find'], stdout=subprocess.PIPE, stderr=subprocess.STDOUT, input='hello', text=True)
```
Comment (Nickolaim) [inline collector.py:18]: I have not tried this code, but please note that subprocess is not running btmgmt. it runs program sudo that runs btmgmt. So sudo needs to pass input to the calling process. It's probably happening.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+RUN addgroup -S hubibtpresence && adduser -S hubibtpresence -G hubibtpresence
+
+# The app relies on btmgmt (bluez-btmgmt package) that needs root privileges (sudo).
+RUN apk add --no-cache sudo bluez-btmgmt
+RUN echo '%hubibtpresence  ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
+
+USER hubibtpresence
+
+WORKDIR /app
```
Comment (Nickolaim) [inline Dockerfile:11]: /app as an env. variable?

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+# set environment variables
```
Comment (Nickolaim) [inline Dockerfile:13]: This line has no value

File: README.md [hunk 4]
```diff
@@ -87,9 +110,23 @@ Note: RSSI is not only affected by distance, but also by the environment (presen
...
-* `pip3 install -r requirements.txt`
-* `python3 main.py`
+
+```
+sudo docker run -d \
+  --name hubibtpresence \
+  --restart always \
+  -v path_to_your/config.yaml:/app/config.yaml \
+  --privileged --net=host \
+  name_of_image
```
Comment (Nickolaim) [inline README.md:124]: image_name?

**Review by Nickolaim — APPROVED** [review]

File: collector.py (hunk 1 above)
Comment (vdbg) [reply collector.py:18]: yes it is happening :)

File: Dockerfile (hunk 3 above)
Comment (vdbg) [reply Dockerfile:13]: I didn't understand the comment

File: README.md (hunk 4 above)
Comment (vdbg) [reply README.md]: ok

File: Dockerfile (hunk 2 above)
Comment (vdbg) [reply Dockerfile:11]: Why? Just being curious

File: Dockerfile (hunk 3 above)
Comment (vdbg) [reply Dockerfile:13]: You mean remove the comment line? ok

**Pull request merged**